user-guide/rate-limiting-tutorial.md

```diff
@@ -112,26 +112,26 @@ spec:
           omit_if_not_present: true
 ```
 
-### v0 API
+### v2 API
```

Comment:
Leave this section alone since this is referencing the old api.

In my opinion this should be removed but that is another problem